Plugins/Mono/Mono_Handlers.cpp

```diff
@@ -244,7 +244,11 @@ MonoString* StackPopString()
     }
 
     LOG_DEBUG("Popped string '%s'.", value.m_sString);
-    return mono_string_new(g_Domain, value.m_sString);
+
+    if (value.m_sString != nullptr)
+        return mono_string_new(g_Domain, ISO88959ToUTF8(value.m_sString).c_str());
```

Comment:
In the interest of brevity:
`return mono_string_new(g_Domain, ISO88959ToUtf(value.CStr()).c_str());`

CExoString::CStr() does the check and returns `""` in case of `nullptr`. This is purely aesthetics, so don't worry about it, but I expect @Liareth will require a few more aesthetic changes so good time to sneak this one in :)